src/components/context-menu/ContextSubMenuItem.svelte

```diff
@@ -28,14 +29,15 @@
   }
 </script>
 
-<div bind:this={div}>
+<div bind:this={div} class="context-sub-menu-item-container">
```

Comment:
Is this class needed? It doesn't look like there's an accompanying style for it.

Reply:
Good catch!